packages/components/header/template.njk

```diff
@@ -32,138 +27,60 @@
         {%- endif %}
       </a>
       {%- else -%}
-      <a class="nhsuk-header__link{% if params.service %} nhsuk-header__link--service {% endif %}" href="{% if params.homeHref %}{{ params.homeHref }}{% else %}/{% endif %}" aria-label="{% if params.ariaLabel %}{{ params.ariaLabel }}{% else %}NHS homepage{% endif %}">
+      <a class="nhsuk-header__link" href="{% if params.homeHref %}{{ params.homeHref }}{% else %}/{% endif %}" aria-label="{% if params.ariaLabel %}{{ params.ariaLabel }}{% else %}NHS homepage{% endif %}">
         {{ nhsLogo | safe }}
-        {%- if params.service %}
-        <span class="nhsuk-header__service-name">
-          {{ params.service.name }}
-        </span>
-        {%- endif %}
       </a>
       {%- endif %}
-    </div>
-    
-
-    {%- if showNav == "false" and showSearch == "false"%}
-
-      {%- if params.transactionalService%}
-        <div class="nhsuk-header__transactional-service-name">
-          <a class="nhsuk-header__transactional-service-name--link" href="{% if params.transactionalService.href %}{{ params.transactionalService.href }}{% else %}/{% endif %}">{{ params.transactionalService.name }}</a>
-        </div>
-        
+      {%- if params.service.name %}
+        <a class="nhsuk-header__service-link" href="{{ params.service.href | default('/') }}">{{ params.service.name }}</a>
```

Comment:
Defaulting to `/` feels safer here (should generally work for most services!) but again it might be worth having an option for no link at all, especially as the service name doesn't visually _look_ like a link, and perhaps for some simple transactional services, a link isn't needed?